bulletin/index.html

```diff
@@ -63,9 +63,9 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
 		<p class="primary-nav--banner-vol">Volume No. 1</p>
 	</div>
 	
-	<div class="bupage">
-		<h1 class="bupage-hero--title">SUMMER ACADEMY: 2007 &#38; 2008</h1>
-		<h2 class="bupage-hero--subtitle">Advancing engineering education in less-advantaged communities</h2> 
+	<div class="bupage" itemscope itemtype="http://schema.org/CreativeWork">
```

Comment:
Why is CreativeWork the most approapriate scheme for the bulletin? I'm not saying it is wrong, I am curious as to why you felt it was the best option.

Reply:
We talked about it. ++ in agreement with your thinking. Thanks!